docs/en/ingest-management/security/certificates.asciidoc

```diff
@@ -0,0 +1,216 @@
+[[secure-connections]]
+[role="xpack"]
+= Encrypt traffic in a self-managed cluster
+
+If you're running a self-managed cluster, configure Transport Layer Security
+(TLS) to encrypt traffic between {agent}s, {fleet-server}, and other components
+in the {stack}.
+
+TIP: Our {ess-product}[hosted {ess}] on {ecloud} provides secure, encrypted
+connections out of the box!
+
+[discrete]
+[[prereqs]]
+== Prerequisites
+
+Configure security and generate certificates for the {stack}:
+
+. {ref}/security-minimal-setup.html[Set up minimal security for {es}]
+. {ref}/security-basic-setup.html[Set up basic security for the {stack}]
+. {ref}/security-basic-setup-https.html[Set up basic security for the {stack} plus secured HTTPS traffic]
```

Comment:
@dedemorton I think kibana security is not required, just Elasticsearch?

Reply:
That's a good point. I was trying to set up all the things in the order that customers will probably do things, but it's probably better to keep things simple.